config.json

```diff
@@ -212,6 +175,17 @@
         "pattern_recognition"
       ]
     },
+    {
+      "slug": "pig-latin",
+      "uuid": "a002e3db-8e9c-4cbf-b00f-f2090bae5d5a",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 6,
+      "topics": [
+        "strings",
+        "transforming"
+      ]
+    },
     {
       "slug": "tree-building",
```

Comment:
This is a refactoring exercise, so personally I'd be happy to see it removed. I'm not sure if we had consensus on removing refactoring exercises though. Definitely the right move to put this at the very end.

Reply:
Yeah, I like refactoring exercises but they're a real pain in the butt to mentor, because refactoring is such a skill in and of itself. We could argue that by the end of the track you should have a decent feel for what idiomatic code should look like in the language... but also, I'm not sure how much I believe that yet :)

Let's keep an eye on it for a few weeks and move it out of core if we still think it's a pain where it is now.